### ISSUE TITLE
Understanding for 1.4.10 and mobile browsers - absolve content authors ("not applicable")

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1483]
https://www.w3.org/WAI/WCAG21/Understanding/reflow.html contains the section explaining that mobile/tablet browsers don't generally do reflow, and that this is a user agent issue. It would likely be good, in the context of how an auditor is supposed to proceed here when evaluating web content on mobile/tablet, to state clearly that the SC is effectively "not applicable" since in practically all mobile/tablet browsers reflow is simply not a thing.

**Comment by patrickhlauke** [comment]
as i seem to have kept part of my thinking firmly in my head, rather than writing it out fully here...

I guess my point is that zooming (or even other methods of resizing content/text, with some interesting recent insights here https://webaim.org/discussion/mail_message?id=45444) in mobile/tablet browsers does not change the CSS viewport dimensions, so the SC in essence cannot be properly/fully tested on mobile/tablet devices unless their viewport happens to hit exactly then normative 320 CSS px width / 256 CSS px height values (which is what normatively is currently defined as the scope of the SC). even most smartphones these days have a default smallest edge (width when in portrait, height when in landscape) that is generally above 320 CSS px / 256 CSS px respectively, and the user has no way to change that directly (because even if they zoomed, the CSS px dimensions remain the same and bidirectional scrollbars appear, and authors can't force this not to happen and to instead cause a change in CSS px dimensions of the viewport only and cause reflow)

**Comment by alastc** [comment]
I'd be happy to say that it is not wise to test on a mobile device.

The iOS/Safari zoom does act like reflow though, it does affect the pixel dimensions. It is not dynamic (i.e. per site), but it is a setting available that it equivalent to zoom.

E.g. Mine is set to 115%, which means my portrait width is 326px. Not suitable for testing, but generally sites which pass will work well.

**Comment by jake-abma** [comment]
Please note this in not always true, we have the portrait / landscape mode where reflow does play a part if the MQs for both different orientations are defined.

**Comment by alastc** [comment]
@jake-abma I'm not sure which bit 'this' is?

**Comment by jake-abma** [comment]
it was this bit: " to state clearly that the SC is effectively "not applicable" since in practically all mobile/tablet browsers reflow is simply not a thing.:"

Maybe a bit too soon as Patrick also added another comment. But just in case that the mentioned sentence is not a fact per se and should not be mentioned as so, I think (the SC is applicable when turning your device...)

**Comment by beustace-nzta** [comment]
@patrickhlauke Curious to know if there's any change in answers to this with phone and browser development current state May 2023? I'm researching how we might test/what applies from WCAG in mobile device testing for web and looking at zoom methods. Found the thread discussion on WebAim.

**Comment by scottaohara** [comment]
closing as resolved per the merging of #4055.

In the updated understanding doc it's called out that [Not all user agents will support this SC's normative sizing requirements](https://deploy-preview-4055--wcag2.netlify.app/understanding/reflow#:~:text=Not%20all%20user%20agents%20will%20support%20this%20SC%27s%20normative%20sizing%20requirements)

this is also in line with the new content added to the latest publication of wcag2ict
